lib/fluent/plugin/in_tail.rb

```diff
@@ -802,6 +771,4 @@ def read_inode
       end
     end
   end
-
-  NewTailInput = TailInput # for backward compatibility
```

Comment:
We can remove it.
in_tail_ex and in_tail_multiline are deprecated 2+ years ago.
New user, v0.14 user, never need these plugin support.